src/porta.rs

```diff
@@ -0,0 +1,207 @@
+//! The Porta Cipher is a polyalphabetic substitution cipher. It was invented
+//! by Giovanni Battista della Porta, an Italian polymath, in 1563.
+//!
+//! # Algorithm
```

Comment:
Overall this is really great documentation. To make sure it's in line with the other ciphers could you please:

1) Drop the sub-headings and make the content under the `Encryption` sub-heading as a new paragraph.
2) Within the above paragraph, specify that this cipher is a reciprocal cipher (process to encrypt is same as decrypt).
3) Follow this paragraph with the substitution table - include a short sentence explaining what it is.
3) Delete the `Decryption` sub-heading.
4) Move the content in the `Encryption Example` section, to the documentation for the `encrypt` function.